med-assist-backend/src/main/java/inc/evil/medassist/appointment/web/AppointmentResponse.java

```diff
@@ -0,0 +1,34 @@
+package inc.evil.medassist.appointment.web;
+
+import inc.evil.medassist.appointment.model.Appointment;
+import inc.evil.medassist.doctor.web.DoctorResponse;
+import inc.evil.medassist.patient.web.PatientResponse;
+import lombok.*;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
+public class AppointmentResponse {
+    private String id;
+    private String appointmentDate;
+    private String startTime;
+    private String endTime;
+    private String operation;
+    private DoctorResponse doctor;
```

Comment:
Here I'm still thinking if `DoctorResponse` is appropriate. We return some unnecessary fields this way

Reply:
Let it be there for now, it is better to return more, than then to request or find ourselves that we miss something